src/Modules/StatCan.OrchardCore.Tabulator/Recipes/Tabulator.recipe.json

```diff
@@ -0,0 +1,115 @@
+{
+    "name": "Table Creator",
+    "displayName": "Table Creator",
+    "description": "Creates tables from a JSON formatted data.",
+    "author": "StatCan Digital Innovation",
+    "website": "digital.statcan.gc.ca",
+    "version": "1.0.0",
+    "issetuprecipe": false,
+    "categories": [],
+    "tags": [], 
+    "steps": [
+        {
+            "name": "feature",
+            "disable": [],
+            "enable": [
+                "StatCan.OrchardCore.Tabulator"
+            ]
+        },
+        {
+          "name": "ContentDefinition",
+          "ContentTypes": [
+            {
+              "Name": "TableCreator",
+              "DisplayName": "TableCreator",
+              "Settings": {
+                "ContentTypeSettings": {
+                  "Creatable": true,
```

Comment:
Widgets usually have the following settings set to `false`: Creatable, Listable, Draftable, Versionable, Securable. Simply change them to `false`.